pkg/liveshare/port_forwarder_test.go

```diff
@@ -55,10 +69,14 @@ func TestPortForwarderStart(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	go func() {
+		notif := <-sendNotification
+		_, _ = notif.conn.DispatchCall(context.Background(), "serverSharing.sharingSucceeded", notif)
+	}()
+
 	done := make(chan error)
 	go func() {
-		const name, remote = "ssh", 8000
-		done <- NewPortForwarder(session, name, remote, false).ForwardToListener(ctx, listen)
+		done <- NewPortForwarder(session, "ssh", port, false).ForwardToListener(ctx, listen)
```

Comment:
There's a preexisting bug in the 'done' channel: it will be sent to exactly twice, and read exactly once. The second send will block forever and its operand ignored, even if it's an error.

Reply:
Added change so both go routines are able to send